src/options.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="options.css">
+  </head>
+	<body>
+		<header>
+      <h1 data-i18n="facebookContainer" class="i18n-string"></h1>
+    </header>
+    <main>
+      <section class="settings-list">
+        <h2 class="i18n-string" data-i18n="fbcSettingsTitle"></h2>
+        <ul>
+          <li>
+            <div class="setting-item">
+              <input class="settings-checkbox" id="allowInstagram" type="checkbox" name="instagram" value="">
+              <div class="checkbox-copy">
+                <label for="allowInstagram">Allow Instagram to load outside of the Facebook container</label>
+                <p>This setting allows for embedded Instagram posts to load on websites outside of the Facebook Container. Note that enabling this setting may allow Facebook to track you across other websites.</p>
+              </div>
+            </div>
+          </li>
+          <li>
+            <div class="setting-item">
+              <input class="settings-checkbox" id="hideBadgeContent" type="checkbox" name="hideBadgeContent" value="">
+              <div class="checkbox-copy">
+                <label for="hideBadgeContent"><img class="inline-fbc-icon" src="fbc-icon.svg" alt=""> Hide FBC Badge </label>
+                <p>This setting hides the badge from appearing (including on hover) on Share/Login elements across non-Facebook sites.</p>
```

Comment:
Add additional copy explaining that sites can still be added to the FB container via the panel